starter-code/css/main.css

```diff
@@ -6,3 +6,166 @@ Green: #00B172
 White: #FFF
 
 */
+@import url("https://fonts.googleapis.com/css?family=PT Sans");
+
+body {
+  font-family: "PT Sans";
+  border: 1px solid #f0f2f5;
+  margin: 10px;
+}
```

Comment:
In this case I wouldn´t have use the body to style as I can see you had a lot of problems with margins and the borders